### PR TITLE
Add Tidelift management docs and blurb on README, as discussed in the ML

### PR DESCRIPTION
**Pull request by nicoddemus**
As discussed in the ML, adding a document describing how we plan to split the funds, and how maintainers can start/stop receiving funding.

A quick draft, so suggestions on the docs are very welcome!

@asottile and @blueyed can already start their subscription process. Payments occur at the end of each month.

@RonnyPfannschmidt would you also like to join right now?

---

I created @pytest-dev/tidelift-admins to list the people which can make changes to Tidelift. What do you folks think?

cc @kszu

### CODE REVIEW

File: TIDELIFT.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+Tidelift aims to make OpenSource sustainable by offering subscription to companies which rely
+on open source packages. This subscription allows it to pay maintainers of those Open Source
+packages to aid sustainability of the work.
+
+Funds
+=====
+
+It was decided in the `mailing list`_ that the Tidelift contribution will be split evenly between
+members of the `contributors team`_ interested in receiving funding.
```
Comment (nicoddemus) [inline TIDELIFT.rst]: @flub in the ML you said "contributors". Did you mean [contributors](https://github.com/orgs/pytest-dev/teams/contributors/members) or [core contributors](https://github.com/orgs/pytest-dev/teams/core/members)? Just to make sure I'm getting this right. 😁

File: TIDELIFT.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+pytest is a member of `Tidelift`_. This document describes how the core team manages
+Tidelift-related activities.
+
+What is it
+==========
+
+Tidelift aims to make OpenSource sustainable by offering subscription to companies which rely
+on open source packages. This subscription allows it to pay maintainers of those Open Source
+packages to aid sustainability of the work.
```
[the remark this answers is not in the document]
Comment (nicoddemus) [reply TIDELIFT.rst]: Oops! Fixed, using "Open Source" always.

File: TIDELIFT.rst [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+Tidelift
+========
+
+pytest is a member of `Tidelift`_. This document describes how the core team manages
+Tidelift-related activities.
+
+What is it
+==========
+
+Tidelift aims to make OpenSource sustainable by offering subscription to companies which rely
```
[the remark this answers is not in the document]
Comment (nicoddemus) [reply TIDELIFT.rst]: Thanks, fixed.

**Comment by RonnyPfannschmidt** [comment]
@nicoddemus thanks for preparing all this, i will add myself to the admins once i can commit to that role

**Comment by nicoddemus** [comment]
@RonnyPfannschmidt thanks, but I meant if you would like to also receive funds right now. In my opinion you should, even if you are in a hiatus right now.

**Comment by RonnyPfannschmidt** [comment]
Thanks @nicoddemus , i appreciate the offer/sentiment!

*But* i believe that we should have a very clear cut and pedantic line  for the fairness that people who pay those funds will perceive.

right now my  position/role is one that shouldn't be paid, so i don't get pay ^^

it is critical to consider, as those that provide the money do not share the nostalgia and gratefulness we feel for each other,
 so from their perception something we do in good faith may look like misuse

i want to take part in ensuring that the overall reality is that pytest is grateful for the funds and makes good and effective use of them

naturally resulting from that mindset is that right now i don't get a  dime :)

File: TIDELIFT.rst (hunk 1 above)
Comment (flub) [reply TIDELIFT.rst]: This is what i meant :)

**Comment by flub** [comment]
the tidelift-admins team also seems like a good call +1 to all this

**Comment by nicoddemus** [comment]
Great, thanks everyone!

@asottile @blueyed, make sure to subscribe to Tidelift so you can get your share by the end of the month! 😁 

cc @pytest-dev/core

**Comment by blueyed** [comment]
btw: just found this (a script to upload release notes): https://github.com/nedbat/coveragepy/commit/e6bd510

**Comment by pfctdayelise** [comment]
Are these the only two docs pages not linked in Contents, somehow?

AKA how tf do I find this Tidelift page on the website?!

**Comment by nicoddemus** [comment]
@pfctdayelise indeed my bad, forgot to add a link in the contents page. :/

**Comment by nicoddemus** [comment]
PR open: https://github.com/pytest-dev/pytest/pull/5297

**Pull request merged**